docs/04-good_documentation.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "What is good documentation"
+output: html_document
+---
+
+# What does good documentation look like?
+
+## Learning Objectives
+
+This chapter will demonstrate how to:  
+
+- Identify major components common to good documentation
+- Describe the purposes of components of good documentation 
+
+## Major components of good documentation
+
+In this chapter we are going to cover the major aspects of a well-documented tool. 
+In the subsequent chapters, we will talk about each of these components in more detail; providing relevant examples and tools. 
+
+![](https://docs.google.com/presentation/d/1cd434bkLer_CJ04GzpsZwzeEA9gjc5Ho6QimiHPbyEg/export/png?id=1cd434bkLer_CJ04GzpsZwzeEA9gjc5Ho6QimiHPbyEg&pageid=gcd0e3791ab_0_23)
+^[For all cartoons:     
+Avataars by https://getavataaars.com/.   
+Icons by https://thenounproject.com/ License CC BY-NC-ND 2.0.     
+Emojis by OpenMoji License: CC BY-SA 4.0.]
+
+### The 'Why' <img src="resources/images/why.png" width="8%">
```

Comment:
Should we use `![]()`?

Reply:
The width adjustments with `![]()` do not seem to work in bookdown. 😞